Tecchly/DLOplanTool
# Dlo-30

### **Related Issue/Keyword:**

Feedback left on your project can be viewed

### **Description:**

Feedback left on a shared project can now be viewed on the 'Feedback' screen. Loading the feedback is a bit slow right now ...

### **Testing:**

**Steps for manual testing:**

1. Share a project with yourself, view it and commend an idea. Take note of the commend type. 
2. Go to the feedback page, wait a bit, and then your feedback should show correctly. 
3. Check that the commend time and type is ok. 
4. Get someone to leave feedback on one of your shared projects and check the above. 

### **Checklist:**

- [] Latest master merged/rebased into your feature branch 
- [] Meets the projects coding conventions
- [] No out of scope changes
- [] #Mentioned other relevant issues
- [] No failure when running the application (obviously lol)
